includes/Checker/Default_Check_Repository.php

```diff
@@ -7,105 +7,59 @@
 
 namespace WordPress\Plugin_Check\Checker;
 
-use Exception;
-
 /**
  * Default Check Repository class.
  *
  * @since n.e.x.t
  */
-class Default_Check_Repository implements Check_Repository {
-
-	/**
-	 * Array map holding all runtime checks.
-	 *
-	 * @since n.e.x.t
-	 * @var array
-	 */
-	protected $runtime_checks = array();
-
-	/**
-	 * Array map holding all static checks.
-	 *
-	 * @since n.e.x.t
-	 * @var array
-	 */
-	protected $static_checks = array();
+class Default_Check_Repository extends Empty_Check_Repository implements Check_Repository {
```

Comment:
The `implements` here is redundant as `Empty_Check_Repository` already implements this interface anyway.
```suggestion
class Default_Check_Repository extends Empty_Check_Repository {
```